docker/run.sh

```diff
@@ -39,17 +39,31 @@ PGPASSWORD=$DB_PASSWORD psql "sslmode=verify-ca sslrootcert=/root/.postgresql/ro
 -c "\copy (SELECT row_to_json(t) FROM (SELECT id,qid,caze_case_id as case_id FROM casev2.uac_qid_link where last_updated >= '$START_OF_PERIOD' and last_updated < '$END_OF_PERIOD') t) To '$QID_FILE';"
 
 
-echo "zipping uac_qid_link file"
+if [ -z "$DATAEXPORT_MI_BUCKET_NAME" ]
```

Comment:
Not sure why we're only checking these here, should the entire script not refuse to run if _both_ these buckets are not set at the start?

Reply:
Yes but is there a likelihood of that?

Reply:
It would be a good deal less code that checking the variables every time we use them? 
What's the intention, is the script supposed to be able to run only pointing at one of the buckets? I'm just not sure why it's checking the vars every step.

Reply:
the logic should be:

export data from DB to file

if NEED_TO_EXPORT_TO_MI then
  copy data file to MI bucket
endif

if NEED_TO_EXPORT_TO_DAP then
  zip data file
  create manifest
  copy zip file to bucket
  copy manifest file to bucket
  delete zip file
  delete manifest file
endif

delete data file

----------------

we don't need all the "I'm not doing stuff" echoing - unuseful code bloat